pages/htmlresources.md

```diff
@@ -216,12 +216,15 @@ Now delete your dummy resource by clicking `Delete`. Then click `Open` on you ht
 
 Finally, push your app to the nation by clicking `Add to Nation`.
 
+_Note:_
+Friendly reminder again to use Firefox when doing these steps or 'Add to Nation' may not appear in other browsers.
+
```

Comment:
Looks having the same issue as #1135. @alexkarasik are you using some text editor with remove trailing whitespace setting on?